WooCommerce/Classes/Analytics/ReactNativeAnalyticsProvider.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+import protocol WooCommerceShared.WCRNAnalyticsProvider
+
+/// Class that provides an interface to `ReactNative` to log track events using the main app `analyticsProvider`.
+///
+class ReactNativeAnalyticsProvider: WCRNAnalyticsProvider {
```

Comment:
nit: can this be marked as `final` or do we plan to subclass this in the future?